war/.jshintrc.json

```diff
@@ -0,0 +1,11 @@
+{
```

Comment:
While we are piling on wishlist items, can you switch to eslint because... well i have actually no solid reasons, i just like it a lot better and it has way more configuration options. https://github.com/jenkins-infra/plugin-site/blob/master/.eslintrc.js as an example

Reply:
I'd also suggest going for eslint. My concern would be that there would suddenly be 100s of new lint warnings. From what I see I would also need to disable all styling rules. 

Do you think there would be any problem in this regard?

Reply:
Just set everything to warn to start. When fix the warnings in a future pr

Reply:
@halkeye is there any eslint preset you recommend that can be good fit this project. I have only used the react-app one for the past year or so (which I think it's quite good).

Reply:
I don't have strong opinions about it. But if you run eslint --init you can infer all the rules based on the current codebase, so would allow things to be linted without making changes

Reply:
I have enabled eslint and stylelint with default configs